### ISSUE TITLE
Can you provide a pre-trained loss curve chart，and sft loss curve chart

### ISSUE THREAD

**Issue by feiyilicare** [PKU-YuanGroup/Chat-UniVi#22]

**Comment by jpthu17** [comment]
Unfortunately, I inadvertently deleted the original training log. I will retrain the model and ensure a new training log is provided.

**Comment by jpthu17** [comment]
This is the loss of the pre-training stage:

[wandb_export_2024-02-25T11_37_23.508+08_00.csv](https://github.com/PKU-YuanGroup/Chat-UniVi/files/14395128/wandb_export_2024-02-25T11_37_23.508%2B08_00.csv)

![image](https://github.com/PKU-YuanGroup/Chat-UniVi/assets/53246557/ea43c1ab-11a7-4380-8971-c267e4271eb6)

**Comment by jpthu17** [comment]
This is the loss of the sft stage:

[wandb_export_2024-02-25T11_40_48.342+08_00.csv](https://github.com/PKU-YuanGroup/Chat-UniVi/files/14395134/wandb_export_2024-02-25T11_40_48.342%2B08_00.csv)

![image](https://github.com/PKU-YuanGroup/Chat-UniVi/assets/53246557/09b67907-e3c1-4c76-a292-5fe8a575d6b5)
